8.counter.py

```diff
@@ -0,0 +1,10 @@
+def counter(text):
+
+    # We use list comprehension with isupper and islower to split lower and upper letters into two lists.
+    s_upper = [i for i in text if i.isupper()]
+    s_lower = [i for i in text if i.islower()]
+    return f"smaller letter: {len(s_upper)}\nupper letter: {len(s_lower)}"
+    # We find the number of lower and upper letters with len function.

```

Comment:
You have to change code snippet `return f"smaller letter: {len(s_upper)}\nupper letter: {len(s_lower)}`. smaller <==>upper ok.